frontend/packages/core/src/AppProvider/short-link-state-hydrator.tsx

```diff
@@ -103,7 +105,7 @@ const ShortLinkStateHydrator = ({
       {state.fromShortLink && (
         <Grid container direction="column" alignItems="flex-end">
           <StyledAlert severity="warning">
-            Loaded shared state. Any local changes will not be preserved.
+            Loaded shared state &quot;{state.hash}&quot;. Any local changes will not be preserved.
```

Comment:
can you make it a link so people can easily right click, copy link address?

Reply:
Sure, let me update it